docs/manual/creating-a-checker.tex

```diff
@@ -2025,6 +2025,9 @@
 
 \item \code{-AdumpOnErrors}: Outputs a stack trace when reporting errors or warnings.
 
+\item \code{-AexceptionLineSeparator}:Controls what line separator is used when printing exceptions.
+This is useful when tools, such as Maven, do not display a full exception messsage.
+
```

Comment:
It would be nice to show a concrete usage.  For example, with an empty string or a semicolon or something?